README.md

```diff
@@ -84,7 +84,7 @@ r := render.New(render.Options{
     AssetNames: func() []string { // Return a list of asset names for the Asset function
       return []string{"filename.tmpl"}
     },
-    Layout: "layout", // Specify a layout template. Layouts can call {{ yield }} to render the current template.
+    Layout: "layout", // Specify a layout template. Layouts can call {{ yield "blockname" }} to render a block in the current template.
```

Comment:
This needs to change again as well. Something like:
`// Specify a layout template. Layouts can call {{ yield }} to render a block in the current template, or call {{ block "css" }} to render a template block.`